jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramPresenter.java

```diff
@@ -269,4 +299,22 @@ public IsWidget getView() {
     public void setProcessService(final Caller<ProcessRuntimeDataService> processService) {
         this.processService = processService;
     }
+
+    public void onProcessDiagramExpandEvent(@Observes ProcessDiagramExpandEvent event) {
```

Comment:
@bxf12315 process expand is not a crucial item to this request, please remove it and create a specific JIRA for that to be handled in the next release.